src/scripts/clipperUI/components/previewViewer/pdfPreview.tsx

```diff
@@ -22,18 +22,35 @@ import { PreviewViewerPdfHeader } from "./previewViewerPdfHeader";
 import * as _ from "lodash";
 
 interface PdfPreviewState {
-	fadeIn: boolean;
+	showPageNumbers: boolean;
 }
 
 class PdfPreview extends PreviewComponentBase<PdfPreviewState, ClipperStateProp> {
-	private static scrollListenerAdded: boolean = false; // done on purpose
+	private static scrollListenerAdded: boolean = false;
+	private static scrollListenerTimeout: number;
 
 	private addScrollListener() {
 		if (!PdfPreview.scrollListenerAdded) {
 			let previewContentContainer = document.getElementById("previewContentContainer");
 			if (!!previewContentContainer) {
+				// When we detect a scroll, show page numbers immediately.
+				// When the user doesn't scroll for some period of time, fade them out.
 				previewContentContainer.addEventListener("scroll", (ev) => {
-					console.log("scroll handler");
+					if (Utils.isNumeric(PdfPreview.scrollListenerTimeout)) {
```

Comment:
You might be able to use lodash's debounce function so you. https://lodash.com/docs/4.16.4#debounce I think it does something similar to what we are trying to do here. 

Otherwise looks good! Thanks Matthew!! I learnt a little bit about how to use css animatinos today